squire_lib/src/ffi_player.rs

```diff
@@ -1 +1,81 @@
+use crate::ffi::{clone_string_to_c_string, FFI_TOURNAMENT_REGISTRY};
+use crate::{
+    identifiers::{PlayerId, PlayerIdentifier, TournamentId},
+    player::{Player, PlayerStatus},
+    tournament::Tournament,
+};
+use std::option::Option;
+use std::os::raw::c_char;
 
+impl PlayerId {
+    /// Returns the player if it can be found in the tournament
+    fn get_tourn_player(self, tid: TournamentId) -> Option<Player> {
+        let tourn: Tournament;
+        unsafe {
+            match FFI_TOURNAMENT_REGISTRY.get_mut().unwrap().get(&tid) {
+                Some(t) => tourn = t.value().clone(),
+                None => {
+                    println!("Cannot find player in get_tourn_player();");
```

Comment:
This print statement is incorrect. The tournament is unable to be found, not the player.

Reply:
00f17b7c64bb77bbeda4f060c47348b9c32df51f